portability-transfer/src/main/java/org/datatransferproject/transfer/JobProcessor.java

```diff
@@ -129,4 +133,15 @@ private AuthData deSerialize(String serialized) throws IOException {
       throw new IOException("Unable to deserialize AuthData", e);
     }
   }
+
+  private void setupConsoleLogger(UUID jobId) {
```

Comment:
I think hard coding the layout/appender is the wrong way to go, it should come from the config file.